invoice_creator.go

```diff
@@ -18,7 +18,20 @@ import (
 
 var byteOrder = binary.BigEndian
 
-const virtualChannel = 12345
+// virtualChannelFromNode deterministically derives a short channel id for
+// multiplexer for a node pubkey.
+func virtualChannelFromNode(key common.PubKey) uint64 {
+	var b [8]byte
+
+	// Take the final bytes of the key.
+	copy(b[:], key[33-8:])
```

Comment:
Would it be better to use `PubKeySize` instead of `33` here?

Reply:
Yes, fixed.